WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardViewModelTests.swift

```diff
@@ -8,7 +8,6 @@ import WordPressAuthenticator
 
 /// Test cases for `BlazeCampaignDashboardViewModel`.
 ///
-@MainActor
```

Comment:
Marking the whole test case with @MainActor causes a warning about concurrency (which will be an error in Swift 6). So I removed this and added the attribute to all the async tests instead.